pywinauto/application.py

```diff
@@ -1247,6 +1248,32 @@ def kill(self):
     # Non PEP-8 aliases
     kill_ = Kill_ = kill
 
+    def is_process_running(self):
+        """
+        Checks that process is still running
+
+        Returns True if process is still running otherwise - False
+        """
+        is_running = False
+        try:
+            h_process = win32api.OpenProcess(
+                win32con.PROCESS_QUERY_INFORMATION,
+                0,
+                self.process)
+            is_running = win32process.GetExitCodeProcess(
+                h_process) == win32defines.PROCESS_STILL_ACTIVE

```

Comment:
Well according to MSDN NULL handle is error result. 

Next GetExitCodeProcess API function will return error saying invalid handle value.

But anyway I can add those tests too